gateways/config.go

```diff
@@ -92,7 +95,7 @@ type EventSourceData struct {
 	// Src contains name of the event source
 	Src string `json:"src"`
 	// Config contains the event source
-	Config string `json:"config"`
```

Comment:
Is this change necessary? We can just set a marshalled string when it's constructed. Or maybe we can create a converting method from `EventSourceData` to `EvenSource` proto and marshal to the string there.